docs/apis-tools/camunda-api-rest/tutorial.md

```diff
@@ -26,7 +26,7 @@ Make sure you keep the generated client credentials in a safe place. The **Clien
 
 If you're interested in how we use a library to handle auth for our code, or to get started, examine the `auth.js` file in the GitHub repository. This file contains a function named `getAccessToken` which executes an OAuth 2.0 protocol to retrieve authentication credentials based on your client ID and client secret. Then, we return the actual token that can be passed as an authorization header in each request.
 
-To set up your credentials, create an `.env` file which will be protected by the `.gitignore` file. You will need to add your `ZEEBE_CLIENT_ID`, `ZEEBE_CLIENT_SECRET`, `ZEEBE_BASE_URL`, and `ZEEBE_AUDIENCE`, which is `zeebe.camunda.io` in a Camunda 8 SaaS environment. For example, your audience may be defined as `ZEEBE_AUDIENCE=zeebe.camunda.io`.
+To set up your credentials, create an `.env` file which will be protected by the `.gitignore` file. You will need to add your `CAMUNDA_CLIENT_ID`, `CAMUNDA_CLIENT_SECRET`, `CAMUNDA_REST_ADDRESS` (represented as `ZEEBE_REST_ADDRESS`), and `CAMUNDA_TOKEN_AUDIENCE` (represented as `ZEEBE_TOKEN_AUDIENCE`), which is `zeebe.camunda.io` in a Camunda 8 SaaS environment. For example, your audience may be defined as `ZEEBE_AUDIENCE=zeebe.camunda.io`.
```

Comment:
Similar to my comment in the related PR, I think this doesn't tell the full story.

> (represented as `ZEEBE_REST_ADDRESS/ZEEBE_TOKEN_AUDIENCE`) 

It is represented as this _in the Console UI_, when you download the .env variables after creating a client. That doesn't come through to me in this message.

Reply:
WDYT about my latest commit?